### ISSUE TITLE
Flutter build apk --release issue

### ISSUE THREAD

**Issue by aakashch0179** [NavidShokoufeh/sstp_flutter#10]
When I try to build release so i found this error 

```
* What went wrong:
Execution failed for task ':sstp_flutter:verifyReleaseResources'.
> A failure occurred while executing com.android.build.gradle.tasks.VerifyLibraryResourcesTask$Action
   > Android resource linking failed
     ERROR:C:\Users\cheta\AppData\Local\Pub\Cache\hosted\pub.dev\sstp_flutter-1.0.9\android\src\main\AndroidManifest.xml:21:6-32:19: AAPT: error: 'specialUse' is incompatible with attribute foregroundServiceType (attr) flags [camera=64, connectedDevice=16, dataSync=1, location=8, mediaPlayback=2, mediaProjection=32, microphone=128, phoneCall=4].

     ERROR:C:\Users\cheta\AppData\Local\Pub\Cache\hosted\pub.dev\sstp_flutter-1.0.9\android\src\main\AndroidManifest.xml:34:10-45:19: AAPT: error: 'specialUse' is incompatible with attribute foregroundServiceType (attr) flags [camera=64, connectedDevice=16, dataSync=1, location=8, mediaPlayback=2, mediaProjection=32, microphone=128, phoneCall=4].
```

**Comment by aakashch0179** [comment]
any update??

**Comment by guibsonmoura** [comment]
To resolve this you need to fix the lines that occurred, notice that in the exception that appears to you it says exactly like this: ERROR:C:\Users\cheta\AppData\Local\Pub\Cache\hosted\pub.dev\sstp_flutter-1.0. 9\android\src\main\AndroidManifest.xml:21:6-32:19: AAPT: error: 'specialUse' is incompatible with attribute foregroundServiceType (attr) flags [camera=64, connectedDevice=16, dataSync=1, location =8, mediaPlayback=2, mediaProjection=32, microphone=128, phoneCall=4]., therefore the problem is in your AndroidManifest.xml, the foregroundServiceType attribute is using the 'specialUse' value which is incompatible and only accepts the flags mentioned above, therefore change the foregroundServiceType flag to connectedDevice in your AndroidManifest.xml and the problem will no longer occur

**Comment by guibsonmoura** [comment]
We need to update this plugin's Android SDK to 34 because, according to this Android Developers page: https://developer.android.com/develop/background-work/services/fg-service-types?hl=pt-br, " In Android 14 (API level 34) and higher, you must declare a suitable service type for each foreground service." That's why when we run flutter build apk --release, it doesn't recognize the "specialUse" attribute and we get the same error as aakashch0179

**Comment by aakashch0179** [comment]
> We need to update this plugin's Android SDK to 34 because, according to this Android Developers page: https://developer.android.com/develop/background-work/services/fg-service-types?hl=pt-br, " In Android 14 (API level 34) and higher, you must declare a suitable service type for each foreground service." That's why when we run flutter build apk --release, it doesn't recognize the "specialUse" attribute and we get the same error as aakashch0179

Do you resolve this issue??

**Comment by NavidShokoufeh** [comment]
Hello everyone , problem has solved on version 1.1.0 . See https://pub.dev/packages/sstp_flutter/changelog#110 
Thanks to dear [Arshia Eihami](https://github.com/blueboy-tm) .

**Comment by NavidShokoufeh** [comment]
I close this issue since it fixed.
